build.gradle

```diff
@@ -202,7 +202,7 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'document-management-client', version: '7.0.0'
 
   compile group: 'uk.gov.hmcts.reform', name: 'core-case-data-store-client', version: '4.7.6'
-  compile group: 'uk.gov.hmcts.reform', name: 'doc-assembly-client', version: '1.07'
+  compile group: 'uk.gov.hmcts.reform', name: 'doc-assembly-client', version: '1.1.0'
```

Comment:
Version 1.1.0 hasn't been released you will need to use:  1.1.0.RC

Ref: https://github.com/hmcts/doc-assembly-client/releases